package.json

```diff
@@ -1,7 +1,7 @@
 {
   "displayName": "Homebridge Midea",
```

Comment:
While we are in package.json lets update...
`"node": "^18.17.0 || ^20.9.0",`
and I think we should change to this as well...
`"displayName": "Homebridge Midea Platform",`

Reply:
The node version string was accepted into the plugin template by the maintainer of that.  See https://github.com/homebridge/homebridge-plugin-template/issues/62